examples/end-to-end-session-based/03-End-to-end-session-based-with-Yoochoose-TF.ipynb

```diff
@@ -0,0 +1,1015 @@
+{
```

Comment:
<p>I guess we don't use the fit_and_evaluate .  Instead, we are using the native tf.keras methods fit()  and <code>evaluate()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/341/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
yes, I fixed that.